app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -43,13 +43,14 @@ export class SearchAndAddOrder extends React.Component {
   handleDiscardOneOrder = (order) => {
     this.props.deleteDraftOrder(order);
     if (order.action === 'DISCONTINUE') {
-      this.setState({ isDelete: false });
+      this.props.setOrderAction('NONE', order.orderNumber);
     }
   }
 
   handleDiscardAllOrders = () => {
     this.setState({ isDelete: false }, () => {
       this.props.deleteAllDraftOrders();
+      this.props.setOrderAction('NONE', '0');
```

Comment:
On line 53, do you need `0` to be a string or would it be better if the value is of type number?

Reply:
Needs to be a string, that particular parameter is order Number which has a data type of string

Reply:
I guess @fuchodeveloper is right here,looking into the other instance call of the method, I noticed the method signature is 
`this.props.setOrderAction(String type, Integer orderNumber);`
However, Javascript is a forgiving language which is not strongly typed but we should bear in mind to follow best practices. A better option would be `this.props.setOrderAction('NONE', 0);`
I hope this explanation clears things further